src/view.js

```diff
@@ -19,6 +19,7 @@ const ClassOptions = [
   'collectionEvents',
   'events',
   'modelEvents',
+  'monitorViewEvents',
```

Comment:
Do we want this option to be set through options?

I think is safer to keep it only in the class definition to avoid using in a class that was created expecting those events to trigger.